README.rst

```diff
@@ -6,7 +6,7 @@
 Atlite
 ======
 
-|PyPI version| |Conda version| |Documentation Status| |travis| |standard-readme compliant|
+|PyPI version| |Conda version| |Documentation Status| |travis| |standard-readme compliant| |GPL-3-or-later-image| |reuse|
```

Comment:
Is the GPL-3-or-later badge appropriate as we have multiple licenses for different aspects of the repo in place?

Reply:
The main code is GPL-3 so I think it's ok.